### PR TITLE
Dividing in components and firebase landingpage from backend display

### PR DESCRIPTION
**Pull request by samyakshah3008**
Divided project into components and added firebase backend and displayed it on Landing page

### CODE REVIEW

File: src/Components/Card.jsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+          <img src={image3} alt="icc-worldcup-trophy" className="card-img" />
+          <div className="card-content">
+            <div className="card-header margin-top center">RCB Fan Quiz</div>
+
+            <div className="card-instruction margin-top text-gray center">
+              Take this quiz to test yourself how well you know about RCB
+            </div>
+            <button className="playnow-btn margin-top">Play now</button>
+          </div>
+        </div> */}
```
Comment (dev-prabhat) [inline src/Components/Card.jsx:61]: Remove commented code

**Review by muskaanshah — APPROVED** [review]

File: src/Components/Card.jsx [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+import { useCategory } from "../contexts/categoryContext";
+
+export default function Card() {
+  const { categories } = useCategory();
+  console.log(categories);
+  return (
+    <div>
+      <div className="quiz-list-container margin-top-xl">
+        {categories.map((category) => (
+          <div className="card">
```
Comment (hinanshisuthar) [inline src/Components/Card.jsx:15]: make sure to have a key

**Review by dev-prabhat — APPROVED** [review]
Suggested some changes after that you can merge this branch, the code structure is good
Looks good to me

File: src/Components/Card.jsx (hunk 1 above)
Comment (hinanshisuthar) [inline src/Components/Card.jsx:61]: Comments can be removed

File: src/Pages/rulepage/rulepage.css [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+    display: flex;
+    flex-direction: column;
+    font-size: 1.5rem;
+    justify-content: center;
+    align-items: center;
+    gap: 1rem;
+    border-radius: 1.5rem;
+}
+
+
```
Comment (hinanshisuthar) [inline src/Pages/rulepage/rulepage.css:29]: make sure to format your code to avoid white spaces.

**Review by hinanshisuthar — COMMENTED** [review]
Good logic implementation. Suggested some changes. Ready to merge.

**Pull request merged**